GliaWidgetsTests/Sources/SerialQueueTests.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import XCTest
+
+@testable import GliaWidgets
+
+final class SerialQueueTests: XCTestCase {
+
+    var queue: SerialQueue!
```

Comment:
To avoid force unwrapping:
```suggestion
    var queue = SerialQueue()
```

Reply:
fixed - c635b8f06962f0641f9fcce4519b20d7c01e0dd5